linebot/api.py

```diff
@@ -61,6 +61,8 @@ def __init__(self, channel_access_token,
             'Authorization': 'Bearer ' + channel_access_token,
             'User-Agent': 'line-bot-sdk-python/' + __version__
         }
+        if retry_key:
+            self.headers['X-Line-Retry-Key'] = retry_key
```

Comment:
[must]
You should not set retry key here because the only 4 APIs are allowed to use the retry key. 

See also https://developers.line.biz/en/docs/messaging-api/retrying-api-request/#designing-how-to-retry-api-requests

> If you include X-Line-Retry-Key in the request header with other APIs than the above, the request will be rejected and the status code 400 will be returned.